lib/elasticity/strategies/alias_index.rb

```diff
@@ -202,7 +202,7 @@ def index_document(type, id, attributes)
         res = @client.index(index: @update_alias, type: type, id: id, body: attributes)
 
         if id = res["_id"]
-          [id, res["created"]]
+          [id, res.dig("_shards", "successful").to_i > 0]
```

Comment:
the `created` key was removed from the API response.

Per the document,  `The index operation is successful in the case successful is at least 1.`
https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-index_.html